modules/data.atmosphere/R/merge.met.variable.R

```diff
@@ -48,9 +48,9 @@ merge_met_variable <- function(in.path,in.prefix,start_date, end_date, merge.fil
   merge.time.attr <- ncdf4::ncatt_get(merge.nc,"time")
   merge.data <- ncdf4::ncvar_get(merge.nc,varid = merge.vars[1])
   
-  udunits2::ud.is.parseable(merge.time.attr$units)
+  units::install_unit(merge.time.attr$units)
```

Comment:
This line wasn't doing anything but fail if line 53 was going to fail, so we can delete it and let the failure happen or not.
```suggestion
```

Reply:
I think this also means `units` can be removed from the PEcan.data.atmosphere Imports.